### PR TITLE
Initial draft of satellite health data structure file.

### LINKED ISSUE #80: Specify data structure for a "satellite health" packet
As per https://github.com/ODU-CGA-CubeSat/flight-software/discussions/39#discussioncomment-1083429, we should specify a [data structure](https://github.com/Mach30/m30ml#data-structures) for a _satellite health_ data packet to be viewed by a ground station operator.

- [ ] should be documented as [data structure](https://github.com/Mach30/m30ml#data-structures) as a `.yaml` file in the `architecture/3-DataStructures/` directory
- [ ] should reference one or more user stories and/or stakeholder needs under `derivedFrom`

### PR DESCRIPTION
**Pull request by kchiu002**
Close #80

### CODE REVIEW

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Satellite Health Data Packet
+purpose: Data structure for satellite health data packet.
+template: |
+  imux: {{imux}}
+  imuy: {{imuy}}
+  imuz: {{imuz}}
+  imuTemperature: {{imuTemperature}}
+  batteryVoltage: {{batteryVoltage}}
```
Comment (jsiciliano33) [inline architecture/3-DataStructures/1-Satellite_Health.yml:8]: Measure of voltage level coming from main battery pack.

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  batteryCurrent: {{batteryCurrent}}
```
Comment (jsiciliano33) [inline architecture/3-DataStructures/1-Satellite_Health.yml:9]: Measure of current (mA) being drawn by the load from the battery pack.

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  batteryBusCurrent: {{batteryBusCurrent}}
+  batteryCurrentDirection: {{batteryCurrentDirection}}
+  batteryTemperature: {{batteryTemperature}}
+  battery12VBusCurrent: {{battery12VBusCurrent}}
```
Comment (jsiciliano33) [inline architecture/3-DataStructures/1-Satellite_Health.yml:13]: Measure of current (mA) being drawn by the load on the 12V bus.

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  battery5VBusCurrent: {{battery5VBusCurrent}}
+  battery3dot3VBusCurrent: {{battery3dot3VBusCurrent}}
```
Comment (jsiciliano33) [inline architecture/3-DataStructures/1-Satellite_Health.yml:15]: Measure of current (mA) being drawn by the load on the 3.3V bus.

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  transmitterCurrent: {{transmitterCurrent}}
```
Comment (jsiciliano33) [inline architecture/3-DataStructures/1-Satellite_Health.yml:16]: Measure of current (mA) being drawn by the radio transmitter during transmission.

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  receiverCurrent: {{receiverCurrent}}
```
Comment (jsiciliano33) [inline architecture/3-DataStructures/1-Satellite_Health.yml:17]: Measure of current (mA) being drawn by the radio receiver during transmission.

**Review by capsulecorplab — COMMENTED** [review]
We should specify the expected data output per IMU axis

File: architecture/3-DataStructures/1-Satellite_Health.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+name: Satellite Health Data Packet
+purpose: Data structure for satellite health data packet.
```
Comment (capsulecorplab) [inline architecture/3-DataStructures/1-Satellite_Health.yml:2]: I'm thinking of appending _from ground station operator's viewpoint_

**Comment by capsulecorplab** [comment]
I'm thinking we should resolve #87 before this PR, as it makes more sense to specify the desired use case (albeit, specified via a user story) prior to specifying a structural/compositional requirement

**Comment by capsulecorplab** [comment]
Need to have conversation w ADS team to confirm units used for IMU (i.e., gyros, magnetometer) & IMU temperature

**Review by capsulecorplab — APPROVED** [review]
LGTM

**Pull request merged**